packages/beacon-node/src/chain/ColumnReconstructionTracker.ts

```diff
@@ -61,9 +59,7 @@ export class ColumnReconstructionTracker {
     // just that it has been triggered for this block root.
     this.running = true;
     this.lastBlockRootHex = blockInput.blockRootHex;
-    const delay =
-      RECONSTRUCTION_DELAY_MIN_MS + Math.random() * (RECONSTRUCTION_DELAY_MAX_MS - RECONSTRUCTION_DELAY_MIN_MS);
-    sleep(delay).then(() => {
+    sleep(delay + Math.random() * RECONSTRUCTION_RANDOM_DELAY_MAX_MS).then(() => {
```

Comment:
I'm in favor of having a mechanism to delay more based on number of columns arrived instead of a random()
similar to [lighthouse](https://github.com/sigp/lighthouse/blob/fd10b632740e441354c4547a12727c29dd288239/beacon_node/network/src/network_beacon_processor/gossip_methods.rs#L1040)
may want to revisit this later